packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -654,4 +710,80 @@ export class PluginLedgerConnectorQuorum
     }
     return this.runDeploy(req);
   }
+
+  // Low level function to call any method from web3.eth
+  // Should be used only if given functionality is not already covered by another endpoint.
+  public async invokeRawWeb3EthMethod(
+    args: InvokeRawWeb3EthMethodV1Request,
+  ): Promise<any> {
+    return new Promise((resolve, rejects) => {
+      this.log.debug("invokeRawWeb3EthMethod input:", JSON.stringify(args));
+
+      Checks.nonBlankString(
+        args.methodName,
+        "web3.eth method string must not be empty",
+      );
+
+      const looseWeb3Eth = this.web3.eth as any;
+      if (
```

Comment:
@outSH If you turn around the if condition's check, you could throw in the negative case and the rest of the logic does not have to be indented at all while the behavior remains the same. (less indentation helps with having less line wraps which in turn increases readability IMO)

Reply:
@petermetz See above